src/platform/nrfconnect/wifi/WiFiManager.h

```diff
@@ -164,6 +164,9 @@ class WiFiManager
         }
     };
 
+    static constexpr uint16_t kRouterSolicitationIntervalMs = 300;
```

Comment:
Based on https://www.rfc-editor.org/rfc/rfc4861#section-6.3.7, we shouldn't send RS more often than `RTR_SOLICITATION_INTERVAL = 4 seconds`. The initial RS delay after the connection establishment should be randomized between 0 and `MAX_RTR_SOLICITATION_DELAY  = 1 second`. Not sure how strict we need to be given that this is the workaround, but fyi that's how the host should behave.

Reply:
I wasn't aware of such restrictions. It's not a big deal so I aligned it. It's always better to be compliant than not.